contract/src/types.js

```diff
@@ -0,0 +1,64 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="ses"/>
+
+/**
+ * @typedef {import('@agoric/swingset-vat/src/vats/network').Bytes} Bytes
+ * @typedef {import('@agoric/swingset-vat/src/vats/network').Data} Data
+ * @typedef {import('@agoric/swingset-vat/src/vats/network').Connection} Connection
+ * @typedef {import('@agoric/swingset-vat/src/vats/network').ConnectionHandler} ConnectionHandler
+ * @typedef {import('@agoric/swingset-vat/src/vats/network').Endpoint} Endpoint
+ */
+
+/**
+ * @template K,V
+ * @typedef {import('@agoric/store').Store<K,V>} Store<K,V>
+ */
+
+/**
+ * @template K,V
+ * @typedef {import('@agoric/weak-store').WeakStore<K,V>} WeakStore<K,V>
+ */
+
+/**
+ * @typedef {string} DenomUri
+ * @typedef {string} Denom
+ * @typedef {string} DepositAddress
+ * @typedef {string} TransferProtocol
+ *
+ * @typedef {Object} Peg
+ * @property {() => string} getAllegedName get the debug name
+ * @property {() => Brand} getLocalBrand get the brand associated with the peg
+ * @property {() => DenomUri} getDenomUri get the denomination identifier
+ *
+ * @typedef {Object} PegDescriptor
+ * @property {Brand} localBrand
+ * @property {DenomUri} denomUri
+ * @property {string} allegedName
```

Comment:
Why do we have both a remotable `Peg` interface type and a `PegDescriptor` record type? Do they carry exactly the same info? IOW, are the getters on the `Peg` object stable?

If we do need both, please list corresponding names in the same order. Or should the `Peg` object just have a single method for obtaining the `PegDescriptor`? Is it a pipelining issue? (If so, nevermind that suggestion)

Reply:
I've moved the `PegDescriptor` into `pegasus.js`, as it is an implementation detail and not a public interface.